test/test.js

```diff
@@ -292,11 +292,59 @@ describe('url-parse', function () {
     assume(parsed.pathname).equals('/foo/bar');
 
     url = 'https:/\\/\\/\\github.com/foo/bar';
+    parsed = parse(url);
     assume(parsed.host).equals('github.com');
     assume(parsed.hostname).equals('github.com');
     assume(parsed.pathname).equals('/foo/bar');
+
+    url = 'https:/github.com/foo/bar';
+    parsed = parse(url);
+    assume(parsed.host).equals('github.com');
+    assume(parsed.pathname).equals('/foo/bar');
+
+    url = 'https:\\github.com/foo/bar';
+    parsed = parse(url);
+    assume(parsed.host).equals('github.com');
+    assume(parsed.pathname).equals('/foo/bar');
+
+    url = 'https:github.com/foo/bar';
+    parsed = parse(url);
+    assume(parsed.host).equals('github.com');
+    assume(parsed.pathname).equals('/foo/bar');
+
+    url = 'https:github.com/foo/bar';
+    parsed = parse(url);
+    assume(parsed.host).equals('github.com');
+    assume(parsed.pathname).equals('/foo/bar');
   });
 
+  it('handles slashes after the protocol for non special URLs', function () {
+    var url = 'foo:example.com'
+      , parsed = parse(url);
+
+    assume(parsed.hostname).equals('');
+    assume(parsed.pathname).equals('example.com');
+    assume(parsed.href).equals('foo:example.com');
+
+    url = 'foo:/example.com';
+    parsed = parse(url);
+    assume(parsed.hostname).equals('');
+    assume(parsed.pathname).equals('/example.com');
+    assume(parsed.href).equals('foo:/example.com');
```

Comment:
Behavior is not consistent with the Node.js WHATWG URL parser if backslashes are used:

```
new URL('foo:\\example.com')
URL {
  href: 'foo:\\example.com',
  origin: 'null',
  protocol: 'foo:',
  username: '',
  password: '',
  host: '',
  hostname: '',
  port: '',
  pathname: '\\example.com',
  search: '',
  searchParams: URLSearchParams {},
  hash: ''
}
```

```
parse('foo:\\example.com')
{
  slashes: false,
  protocol: 'foo:',
  hash: '',
  query: '',
  pathname: '/example.com',
  auth: '',
  host: '',
  port: '',
  hostname: '',
  password: '',
  username: '',
  origin: 'null',
  href: 'foo:/example.com'
}
```

Reply:
Fixed in eda1342 and e24fca3.

Reply:
Fixed in https://github.com/unshiftio/url-parse/pull/209/commits/48bdd73271eceec58b094e92a737b5028d61879a.